zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksum.java

```diff
@@ -42,9 +46,25 @@ public static ImmutableChecksumsSFV read(final Path checksumPath) throws IOExcep
   }
 
   public static MutableChecksumsSFV calculate(final Path snapshotDirectory) throws IOException {
+    return createChecksumForSnapshot(snapshotDirectory, null);
+  }
+
+  public static MutableChecksumsSFV calculateWithFullFileChecksums(
```

Comment:
:wrench: I think the name doesn't quite fit. It's more like `calculateWithProvidedChecksums`, right?